java/src/main/native/clang-format.README

```diff
@@ -0,0 +1,13 @@
+README
```

Comment:
Are you already using clang-format for the Java side? We haven't added it for C++ yet.

Reply:
We were using IntelliJ for formatting java, as it is tied in with a fairly popular IDE, but that could change in the future to all use clang-format. Currently clang-format we used only for the cpp code in the java directory, as we tried to get us to be more cudf friendly. When cudf moves to a standard .clang-format file, the idea is for us to move too.